### PR TITLE
Updated API key instructions for citra update on June 26th

### PR DESCRIPTION
**Pull request by pMaske**
Updated API key links in toolkit samples

### CODE REVIEW

File: src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml [hunk 1]
```diff
@@ -71,8 +71,7 @@
                                 Background="CornflowerBlue" Foreground="White"
                                 Click="SaveApiKey_Click" />
                                 <TextBlock TextWrapping="Wrap" Margin="0,10,0,0">
-                            <Span>You can get or create an API key from <Hyperlink NavigateUri="https://developers.arcgis.com/api-keys/">your developer dashboard</Hyperlink>.</Span><LineBreak></LineBreak>
-                            <Hyperlink NavigateUri="https://developers.arcgis.com/net/security-and-authentication/#api-keys">More info on API Keys</Hyperlink>
+                                  <Span>You can get or create an API key as described in <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key">this tutorial</Hyperlink>.</Span>
```
Comment (dotMorten) [inline src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml:74]: You removed the more-info here and below, but maui still has the more info. Which solution are we going with?

File: src/Samples/Toolkit.SampleApp.Maui/MainPage.xaml.cs [hunk 2]
```diff
@@ -91,11 +91,11 @@ private async void SaveApiKey_Click(object sender, EventArgs e)
...
     private void DashboardLinkTapped(object sender, TappedEventArgs e)
     {
-        Microsoft.Maui.ApplicationModel.Launcher.OpenAsync("https://developers.arcgis.com/api-keys/");
+        Microsoft.Maui.ApplicationModel.Launcher.OpenAsync("https://links.esri.com/create-an-api-key");
     }
 
     private void MoreInfoLinkTapped(object sender, TappedEventArgs e)
     {
-        Microsoft.Maui.ApplicationModel.Launcher.OpenAsync("https://developers.arcgis.com/net/security-and-authentication/#api-keys");
+        Microsoft.Maui.ApplicationModel.Launcher.OpenAsync("https://developers.arcgis.com/net/security-and-authentication/tutorials/create-an-api-key/");
```
[the remark this answers is not in the document]
Comment (pMaske) [reply src/Samples/Toolkit.SampleApp.Maui/MainPage.xaml.cs]: This is no longer needed. Removed in my latest commit.

File: src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml (hunk 1 above)
Comment (pMaske) [reply src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml:74]: More-info is also not needed any longer. I missed to removed it from MAUI. Just updated the PR. Please review the latest commit.

**Review by dotMorten — APPROVED** [review]

File: src/Samples/Toolkit.SampleApp.Maui/MainPage.xaml [hunk 3]
```diff
@@ -36,8 +36,8 @@
                         <Label MaxLines="5" Margin="0,10,0,0" >
                             <Label.FormattedText>
                                 <FormattedString>
-                                    <Span Text="You can get or create an API key from " />
-                                    <Span Text="your developer dashboard" TextDecorations="Underline" TextColor="{AppThemeBinding Light={StaticResource Blue100Accent}, Dark={StaticResource Blue300Accent}}" >
+                                    <Span Text="You can get or create an API key by following" />
```
Comment (maryharvey) [inline src/Samples/Toolkit.SampleApp.Maui/MainPage.xaml:39]: You can create an API Key using the Create an api key Tutorial... but you cannot get/copy one after the API Key has been generated. You only get one opportunity to copy it so I would be tempted to just say "You can create an API Key access token by...."

File: src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml (hunk 1 above)
Comment (maryharvey) [reply src/Samples/Toolkit.SampleApp.UWP/MainPage.xaml:74]: Same comment as above "You can create an API Key access token as described...."

File: src/Samples/Toolkit.SampleApp.WPF/MainWindow.xaml [hunk 4]
```diff
@@ -59,8 +59,7 @@
                                 Click="SaveApiKey_Click"
                                 />
                         <TextBlock TextWrapping="Wrap" Margin="0,10,0,0">
-                            <Span>You can get or create an API key from <Hyperlink NavigateUri="https://developers.arcgis.com/api-keys/" RequestNavigate="Hyperlink_RequestNavigate">your developer dashboard</Hyperlink>.</Span><LineBreak></LineBreak>
-                            <Hyperlink NavigateUri="https://developers.arcgis.com/net/security-and-authentication/#api-keys" RequestNavigate="Hyperlink_RequestNavigate">More info on API Keys</Hyperlink>
+                            <Span>You can get or create an API key as described in <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key" RequestNavigate="Hyperlink_RequestNavigate">this tutorial</Hyperlink>.</Span>
```
Comment (maryharvey) [inline src/Samples/Toolkit.SampleApp.WPF/MainWindow.xaml:62]: As above "You can create an API Key access token as..."

File: src/Samples/Toolkit.SampleApp.WinUI/MainWindow.xaml [hunk 5]
```diff
@@ -64,8 +64,7 @@
                                 Background="CornflowerBlue" Foreground="White"
                                 Click="SaveApiKey_Click" />
                                 <TextBlock TextWrapping="Wrap" Margin="0,10,0,0">
-                            <Span>You can get or create an API key from <Hyperlink NavigateUri="https://developers.arcgis.com/api-keys/">your developer dashboard</Hyperlink>.</Span><LineBreak></LineBreak>
-                            <Hyperlink NavigateUri="https://developers.arcgis.com/net/security-and-authentication/#api-keys">More info on API Keys</Hyperlink>
+                                  <Span>You can get or create an API key as described in <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key">this tutorial</Hyperlink>.</Span>
```
Comment (maryharvey) [inline src/Samples/Toolkit.SampleApp.WinUI/MainWindow.xaml:67]: As above "You can create an API Key access token as..."

**Review by maryharvey — CHANGES_REQUESTED** [review]
@pMaske a small suggestion. thanks

**Comment by pMaske** [comment]
@maryharvey please take another look.

**Review by maryharvey — APPROVED** [review]
@pMaske Looks good.

**Pull request merged**